.gitignore

```diff
@@ -34,4 +34,17 @@
 
 # build
 _build
-antares-deps
\ No newline at end of file
+antares-deps
+
+# Cmake generated files
+CMakeCache.txt
+CMakeFiles/*
+
+# Outputs of instances
+*out.json
+*benderssequentialLog*
+*bendersmpiLog*
+*merge_mpsLog*
+*log_merged*
+*benders_output_trace.csv
+src/src_python/config.yaml
```

Comment:
I pushed README's in folders "outputs" in each Benders test case so that the folder "outputs" is present in the repo. 
If the folder does not exist, Benders fails to write out.json and the test fails.

Reply:
You can have folder "output" indexed, but exclude its contents. See [this SO thread and reply](https://stackoverflow.com/a/932982).

Reply:
Thanks, I changed this.